machine_learning/forecasting/run.py

```diff
@@ -0,0 +1,117 @@
+"""
+this is code for forecasting
+but i modified it and used it for safety checker of data
+for ex: you have a online shop and for some reason some data are
+missing (the amount of data that u expected are not supposed to be)
+        then we can use it
+*ps : 1. ofc we can use normal statistic method but in this case
+         the data is quite absurd and only a little^^
+      2. ofc u can use this and modified it for forecasting purpose
+         for the next 3 months sales or something,
+         u can just adjust it for ur own purpose
+"""
+
+import numpy as np
+import pandas as pd
+from sklearn.preprocessing import Normalizer
+from sklearn.svm import SVR
+from statsmodels.tsa.statespace.sarimax import SARIMAX
+
+
+def lin_reg_pred(train_dt, train_usr, train_mtch, test_dt, test_mtch):
+    # linear regression, return floats
+    x = []
```

Comment:
Let’s use a Python `list compréhension` to put these three lines on one line.

Reply:
yes, i did that but. i have some trouble with spell correction such as:
-white trailing space .. since its keep failed so i changed it back to the start when it passed all test
and since its my first time using pre commit, will try it and use this for learn^^